_data/sidebars/home_sidebar.yml

```diff
@@ -74,6 +74,14 @@ entries:
       url: /widgets/
       output: web
 
+    - title: Plugins & platform-specific code
```

Comment:
No we didn't. "Platform-specific code" is what we started using when we deemed "Native code" too confusing. I'm worried about doing another "indirection".